client/src/components/App.js

```diff
@@ -32,6 +33,9 @@ const App = (props) => {
           <SandwichShow user={currentUser} />
         </Route>
         <Route exact path="/" component={SandwichList} />
+        <Route exact path="/user/page">
+          <UserProfile user={currentUser}/>
+        </Route>
```

Comment:
anyone on the internet can navigate to this page. This should be replaced with the AuthenticatedRoute pattern (see said component in your app) instead. Only logged in users should be able to navigate to this page